exercise/templatetags/exercise.py

```diff
@@ -118,33 +118,34 @@ def submission_status(status):
     return Submission.STATUS[status]
 
 def _get_points_type(context):
-    points_type = set()
+    class_name = ""
 
     if ((context['confirm_the_level']) or
         context['difficulty'] == "-" and context['max'] <= 2):
 
         if (context['full_score'] or
             (context['passed'] and context['confirm_the_level'])):
-            points_type.add("accepted")
+            class_name = "accepted"
         elif context['passed'] and not context['confirm_the_level']:
-            points_type.add("ok")
+            class_name = "ok"
         elif ((context['points'] == 0 and context['submitted'])
             or context['graded']):
-            points_type.add("rejected")
+            class_name = "rejected"
         elif context.get('confirmable_points'):
-            points_type.add("missing")
+            class_name = "missing"
         else:
-            points_type.add("not_submitted")
+            class_name = "not_submitted"
     else: # normal exercise without textual status
         if context['full_score']:
-            points_type.add("full")
+            class_name = "full"
         elif context['passed']:
-            points_type.add("passed")
+            class_name = "passed"
         elif context['submitted']:
-            points_type.add("not_passed")
-    if not context['confirm_the_level'] and context.get('unconfirmed'):
-        points_type.add("unconfirmed")
-    return points_type
+            class_name = "not_passed"
+
+    # the second element in the returned tuple is a boolean indicating
+    # the confimation status of the points (unconfirmed is indicated by True).
+    return (class_name, not context['confirm_the_level'] and context.get('unconfirmed'))
```

Comment:
Bit more clear if temporary variable is used. Variable name documents what is going on and then the last line will be trivial to read and understand :)

Reply:
Done.